output/cloud/output.go

```diff
@@ -341,6 +345,8 @@ func (out *Output) startVersionedOutput() error {
 	}
 	var err error
 
+	out.usage.String("output.cloud.test_run_id", out.testRunID)
```

Comment:
```suggestion
	out.usage.String("cloud/test_run_id", out.testRunID)
```

This likely is the better name